src/tooltip/docs/demo.html

```diff
@@ -20,7 +20,7 @@
       <a href="#" tooltip-animation="false" tooltip="I don't fade. :-(">fading</a>
       at elementum eu, facilisis sed odio morbi quis commodo odio. In cursus
       <a href="#" tooltip-popup-delay='1000' tooltip='appears with delay'>delayed</a> turpis massa tincidunt dui ut.
-      <a href="#" tooltip-template="myTooltipTemplate.html">Custom template</a>
+      <a href="#" tooltip-template="'myTooltipTemplate.html'">Custom template</a>
```

Comment:
This is because if a user needs to use a trusted resource url, the user can now do the following.

In controller:
```
$scope.templateUrl = $sce.trustAsResourceUrl('http://someurl.com/template.htm');
```

In template `tooltip-template="templateUrl"`.

Before, the if the user did `tooltip-template="{{templateUrl}}`, the URL gets converted into a plain string after interpolation and hence, loses its *trusted context*. Then there's no way for the user to specify a trusted URL. This works fine for URLs on the current security origin but will fail when the user specifies URLs from another security origin (different domain/protocol/port).